tests/test_board.py

```diff
@@ -21,21 +21,37 @@ def test_can_occupy_a_space(board, space):
     assert board.get_spaces()[space] == 'x'
 
 
-def test_knows_when_no_spaces_are_occupied(board):
+def test_returns_false_when_no_spaces_are_occupied(board):
```

Comment:
I like the change of names for the tests. I wonder if it's possible to remove any notion of a specific data type?

A contrived example, but if we change the implementation of `is_full()` so that we return `True == 1`  or `False == 0`, is the test's name still accurate and does it still make sense?

If we're focusing solely on behaviour, might something like `test_when_no_spaces_are_occupied_then_board_is_empty()` be a better name?